src/components/recoverPublicKey/useRecoverPublicKey.ts

```diff
@@ -93,6 +93,7 @@ export function useRecoverPublicKey() {
 
   const recoverPublicKey = useCallback(
```

Comment:
I've found that we may be managing other public keys as well, like the user's public key. Considering that it may be best to name this `recipientPublicKey` instead of just `publicKey`. Perhaps in a future PR.

Reply:
Yeah, I acutally had recipientPublicKey typed in a few times and then didn't make the change. Future refactor for sure